src/java.base/unix/native/libnio/ch/DatagramChannelImpl.c

```diff
@@ -50,20 +50,28 @@ Java_sun_nio_ch_DatagramChannelImpl_disconnect0(JNIEnv *env, jclass clazz,
     jint fd = fdval(env, fdo);
     int rv;
 
+#if defined(__APPLE__)
+    // On MacOSX systems we use disconnectx
```

Comment:
Thanks for doing this. Do you mind changing this comment to "macOS" as I think that is how the OS is named these days.

Reply:
Done. I have pushed an update to this PR with this change.